compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/WindowContentLayout.desktop.kt

```diff
@@ -55,8 +55,13 @@ internal fun WindowContentLayout(
 
             val contentPlaceables = buildList(measurables.size) {
                 measurables.fastForEach {
-                    if (it != resizerMeasurable)
-                        add(it.measure(constraints))
+                    if (it != resizerMeasurable) {
```

Comment:
This make the root node sizes take the window size. (Aligning the bahaviour with Android).
But I believe there is no strong reason to do that.

Reply:
This change made some tests fail. I reverted it in the last commit.  I think we don't really need it (the only reason could be is to align with Android, but again, I don't understand why it's done like that there. Asked in Slack).

Reply:
The desktop tests passed after this change was reverted. 
So let's continue without it.

Reply:
Let's remove from the description as well.